include/psa/crypto_config.h

```diff
@@ -80,6 +80,20 @@
 #define PSA_WANT_ALG_TLS12_PSK_TO_MS            1
 #define PSA_WANT_ALG_XTS                        1
 
+#define PSA_WANT_ECC_BP256R1                    1
```

Comment:
Looks like you forgot to update these names, @d3zd3z

Reply:
@stevew817 thanks, fixed.